pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_key_template.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/aead"
+	commonpb "github.com/google/tink/go/proto/common_go_proto"
+	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle"
+	ecdhespb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdhes_aead_go_proto"
+)
+
+// ECDHES256AES256GCMKeyTemplate is a KeyTemplate that generates an ECDH-ES P-256 key wrapping and AES256-GCM CEK with
+// the following parameters:
+//  - Key Wrapping: ECDH-ES over NIST P-256
+//  - Content Encryption: AES256-GCM
```

Comment:
also need 25519

Reply:
will handle 25519 when we add chacha key support

Reply:
there's a todo at line 65 below for this

Reply:
will be done as part of #1657